### DISCUSSION TITLE
2023/09/tutorial-dan-tips-untuk-firmware-openwrt

### DISCUSSION THREAD

**Discussion by giscus** [madi10/MANTANKODE#39] [General]
# 2023/09/tutorial-dan-tips-untuk-firmware-openwrt

Rumpi bersama mantankode mengenai error yang ada pada OpenWrt.

https://www.mantankode.net/2023/09/tutorial-dan-tips-untuk-firmware-openwrt.html

**Comment by madi10** [comment]
susah yah komentar pakai github? jarang yang komentar.
